crates/swc/tests/tsc-references/destructuringControlFlow.2.minified.js

```diff
@@ -1,4 +1,4 @@
 //// [destructuringControlFlow.ts]
-(0, [
+[
     "foo"
-][1]).toUpperCase();
+][1].toUpperCase();
```

Comment:
Not that it matters in this specific case, but this has changed the `this` context of the method.

Reply:
Nice catch, I'll add a test case for it.